tinker/static/assets/js/tinker.js

```diff
@@ -14,4 +14,130 @@ $(document).ready(function () {
             $(this).text("- Less Search Filters")
         }
     });
-});
\ No newline at end of file
+});
+
+function searchCookie(prevInputs, type, groups) {
+    let inputs = prevInputs;
+    if (Cookies.get(type + "-search-cookie") != null) {
+        inputs = Cookies.get(type + "-search-cookie");
+    }
+    Cookies.set(type + "-search-cookie", inputs, { expires: 1, path: '/' });
+
+    let viewList = []
+    if (type === "event") {
+        viewList = ['My Events', 'All Events', 'Other Events', 'User Events'];
+        $("#search-box").empty();
+        $("#search-box").append("<label for=\"event-title\">Event Title:</label>\n" +
```

Comment:
Why are you using JS to create HTML snippets? I think it might make more sense to just use JS to modify the values.

Reply:
Fixed